slither.config.json

```diff
@@ -0,0 +1,4 @@
+{
+  "detectors_to_exclude": "incorrect-versions-of-solidity,conformance-to-solidity-naming-conventions",
```

Comment:
We really do not care about solidity versions. Everything above >0.8.7 will throw a warning. Also, namings in this repo are not really straight forward since we are conforming to Chainlink's names through our interface.